contents/ios/IntroToIos.md

```diff
@@ -0,0 +1,103 @@
+
+#### Better compatability and standardisation
+
+iOS applications are specific to only the Apple devices of the iPhone and iPad line, whereas Android runs across numerous different types of devices, including phones, tablets, watches, and more. The large variation of devices poses a compatability problem from a developer's standpoint, because now applications have to be tested and working across all these different devices, and also regularly maintained this way. Each hardware device may possibly be running their own version of Android or will only support up to a particular version, thus Android developers will have to ensure that their application is also compatible across different versions of Android.
+
+With lesser devices to target, iOS developers have more control over their application and are able to customise the application to a greater extent, given that there are less variations (in terms of device compatibility) to worry about. OS versioning in iOS is also more structured, with better backwards-compatability across their devices. Users of iOS devices are always immediately prompted whenever a new version of iOS is available for download, regardless of how old your device is. This means that as developers, we have to worry less about compatability issues, and focus more on the application itself.
+
+#### Lucrative market
+
+While Android has higher market share of devices, iOS applications generate much more revenue than Android, [nearly double](https://techcrunch.com/2018/07/16/apples-app-store-revenue-nearly-double-that-of-google-play-in-first-half-of-2018/). This is largely due to the fact that users of iOS devices are generally more likely to spend on apps, and also because the cost for iOS developers to submit their applicaation to the App Store is higher. 
+
+As an independent iOS developer, this means that your iOS application could bring in more revenue for you as compared to an Android one, assuming you choose to monetize it. 
+
+More app revenue for iOS applications also means companies will look to tackle the iOS platform more so than Android, thus the demand for iOS developers will be high. This in turn results in [higher salary for iOS developers compared to their Android counterparts](https://www.fiercewireless.com/developer/ios-developers-earn-roughly-10k-more-than-android-counterparts-study-shows).
+
+#### Higher quality apps
+
+Publishing an app to the Apple App Store is by no means a simple task. First and foremost, there is an annual fee that you have to pay as a developer in order to [have your app published on the App Store](https://developer.apple.com/ios/submit/). Upon submission, the App Store will then test your app for every possible bug and validation errors before they will proceed to publish on the store. This is drastically different from the Android Play Store where approval process is not as stringent. 
```

Comment:
> Upon submission, the App Store will then test your app for every possible bug and validation errors

This is an interesting point. Are there any articles that can be linked to this for further reading?